content/en/docs/contributing/release-process.md

```diff
@@ -10,6 +10,61 @@ new release of cert-manager.
 
 ## Prerequisites
 
+<!--
+About "alert" and "pageinfo": https://www.docsy.dev/docs/adding-content/shortcodes/#shortcode-helpers
+-->
+
+{{% pageinfo color="warning" %}}
+
+⛔️ Do not proceed with the release process if you do not satisfy with one of the
```

Comment:
> Do not proceed with the release process if you do not meet all of the following conditions:

Reply:
Thanks! Fixed in 2909939 👍

Reply:
Thanks! Fixed in da4b1e94c8380d3d9d9cdabf75f88aaa820564ba